src/signalo/settings.py

```diff
@@ -169,3 +171,11 @@
 
 # Geometry's SRID. This can only be changed prior to initializing the database.
 GEOMETRY_SRID = int(os.environ.get("GEOMETRY_SRID", "2056"))
+
+INTERNAL_IPS = [
+    "127.0.0.1",  # so that Django toolbar is displayed to localhost
+]
+
+DEBUG_TOOLBAR_CONFIG = {
+    "SHOW_TOOLBAR_CALLBACK": lambda request: True,
```

Comment:
Is it on purpose why the callback is returning `True` without any condition? I think it can be boolean instead of lambda in this case, so it's clear the author intents. Otherwise it can be dependent on the current `DEBUG` value?